pkgs/development/interpreters/ruby/rubygems/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  name = "rubygems";
+  version = "3.0.3";
+
+  src = fetchurl {
+    url = "https://rubygems.org/rubygems/rubygems-${version}.tgz";
+    sha256 = "0b6b9ads8522804xv8b8498gqwsv4qawv13f81kyc7g966y7lfmy";
+  };
+
+  patches = [
+    ./0001-add-post-extract-hook.patch
+    ./0002-binaries-with-env-shebang.patch
+    ./0003-gem-install-default-to-user.patch
```

Comment:
in general we try to use fetchPatch when possible to avoid growing the nixpkgs repo unnecessarily

Reply:
IMO "when possible" shouldn't include pointing to patch URL's on a single maintainer's GitHub fork – if this were the case, Nixpkgs wouldn't need to contain any patch files at all, but that's clearly not how things are?

Reply:
To be fair I don't think we have a clear policy on that and it's probably not enforced consistently across nixpkgs.

Patches should be cached by the binary cache so having them on a fork shouldn't really be a problem.

Reply:
Would it be better for you if we maintained the fork in the nix-community org?